TWIR Week 29 (#349)
giantswarm/releases: announcements/2020-Week-29/kvm.md

# :zap: This Week in KVM Releases (July 17, 2020):zap:

[KVM v12.0.0](https://github.com/giantswarm/releases/blob/master/kvm/v12.0.0) is the first Giant Swarm release which includes **Kubernetes v1.17**. Many core components and default apps have been updated for improved reliability and observability.

**Optional Apps Releases**  
(Can be installed from App Catalogs)

> 1. [Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.

Author: ```suggestion
[Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```

> 2. [NGINX IC v1.7.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#173---2020-07-16) fixes a regression introduced in the previous version (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)

Author: ```suggestion
```

# :zap: This Week in KVM Releases (July 17, 2020):zap:

> Please let <!subteam^S0GSG846L> know if you have any feedback or questions by replying to this announcement in a thread.

Reviewer: ```suggestion
Please reply to this announcement in a thread if you have any feedback or questions.
```
PO group handle doesn't look good, remove for now?

Author: sure!
